index.html

```diff
@@ -8,43 +9,43 @@
       <h1>ColoRandom</h1>
       <section class="current-palette">
         <div id="color1">
-          <div class="box" id="box1">
+          <div class="box box-1" id="box1">
           </div>
           <div class="hexcode">
             <p>#214485</p>
-            <img src="./assets/padlock.png">
+            <img class="lock" src="./assets/lock-unlocked.svg">
           </div>
         </div>
         <div id="color2">
-          <div class="box" id="box2">
+          <div class="box box-2" id="box2">
           </div>
           <div class="hexcode">
             <p>#852147</p>
-            <img src="./assets/padlock.png">
+            <img class="lock" src="./assets/lock-unlocked.svg">
           </div>
         </div>
         <div id="color3">
-          <div class="box" id="box3">
+          <div class="box box-3" id="box3">
           </div>
           <div class="hexcode">
             <p>#786F63</p>
-            <img src="./assets/padlock.png">
+            <img class="lock" src="./assets/lock-unlocked.svg">
           </div>
         </div>
         <div id="color4">
-          <div class="box" id="box4">
+          <div class="box box-4" id="box4">
           </div>
           <div class="hexcode">
             <p>#21952B</p>
-            <img src="./assets/padlock.png">
+            <img class="lock" src="./assets/lock-unlocked.svg">
           </div>
         </div>
         <div id="color5">
-          <div class="box" id="box5">
+          <div class="box box-5" id="box5">
           </div>
           <div class="hexcode">
             <p>#123456</p>
-            <img src="./assets/padlock.png">
```

Comment:
I like that we found a way to do this without toggling 'hidden' on and off for two different images.